### PR TITLE
v0.3.2-beta changes

### PR DESCRIPTION
**Pull request by temaniarpit27**
# Description

This PR is reverting the RPC change for eth_getBlock

# Changes

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Checklist

- [x] I have added at least 2 reviewer or the whole pos-v1 team
- [ ] I have added sufficient documentation in code
- [ ] I will be resolving comments - if any - by pushing each fix in a separate commit and linking the commit hash in the comment reply

# Cross repository changes

- [ ] This PR requires changes to heimdall
    - In case link the PR here:
- [ ] This PR requires changes to matic-cli
    - In case link the PR here:

## Testing

- [ ] I have added unit tests
- [ ] I have added tests to CI
- [ ] I have tested this code manually on local environment
- [ ] I have tested this code manually on remote devnet using express-cli
- [ ] I have tested this code manually on mumbai
- [ ] I have created new e2e tests into express-cli

### Manual tests

Please complete this section with the steps you performed if you ran manual tests for this functionality, otherwise delete it

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -29,12 +29,6 @@ jobs:
             TAG=${GITHUB_REF#refs/tags/}
             echo ::set-output name=tag_name::${TAG}
 
-      - name: Login to Docker Hub
```
Comment (cffls) [inline .github/workflows/release.yml:32]: We shouldn't revert changes in this file for go 1.19.

Comment (temaniarpit27) [reply .github/workflows/release.yml]: reverted

**Review by cffls — APPROVED** [review]

File: internal/cli/snapshot.go [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+type SnapshotCommand struct {
+	UI cli.Ui
+}
+
+// MarkDown implements cli.MarkDown interface
+func (a *SnapshotCommand) MarkDown() string {
+	items := []string{
+		"# snapshot",
+		"The ```snapshot``` command groups snapshot related actions:",
+		"- [```snapshot prune-state```](./snapshot_prune-state.md): Joins the local client to another remote peer.",
```
Comment (manav2401) [inline internal/cli/snapshot.go:28]: The description :)

**Review by manav2401 — APPROVED** [review]

File: internal/cli/snapshot.go (hunk 2 above)
Comment (cffls) [reply internal/cli/snapshot.go:28]: Nice catch! Too much copy-paste 😂

File: docs/README.md [hunk 3]
```diff
@@ -11,6 +11,8 @@
   $ bor server <flags>
   ```
 
+  See [here](./cli/server.md) for more flag details.
```
Comment (Amazinggrace07) [inline docs/README.md:14]: 100.101.90.133

**Pull request merged**